### ISSUE TITLE
Number editor breaks min/max validator in Chromium/IE11

### ISSUE THREAD

**Issue by bedaes** [olifolkerd/tabulator#1718]
**Describe the bug**

Setting `editor:"number"` when using a min/max validator leads to `Uncaught RangeError: Maximum call stack size exceeded.` in Chromium. IE simply stops to respond without any error message. This doesn't happen in Firefox though.

```
tabulator.min.js:6 Uncaught RangeError: Maximum call stack size exceeded.
    at w.focusCellNoEvent (tabulator.min.js:6)
    at i (tabulator.min.js:6)
    at s (tabulator.min.js:6)
    at HTMLInputElement.<anonymous> (tabulator.min.js:6)
    at w.focusCellNoEvent (tabulator.min.js:6)
    at i (tabulator.min.js:6)
    at s (tabulator.min.js:6)
    at HTMLInputElement.<anonymous> (tabulator.min.js:6)
    at w.focusCellNoEvent (tabulator.min.js:6)
    at i (tabulator.min.js:6)
```

**Tabulator Info**
- 4.1.5

```
var table = new Tabulator("#example-table", {
    height:205,
    data:[{"id":1}],
    columns:[
        {title: "id", field:"id", editor:"number", validator:[
          { type:"min", parameters:1 },
          { type:"max", parameters:2 }
        ]}
    ]
});
```

**To Reproduce**
1. Go to https://jsfiddle.net/az4s9wk5/2/
2. Type 0 or 3 into the cell
3. See error

**Desktop (please complete the following information):**
 - Chromium 71 / IE11

**Comment by olifolkerd** [comment]
Hey @bedaes 

Thanks for letting me know. i will make sure this is resolved for the 4.2 release at the end of the month.

Cheers

Oli :)

**Comment by olifolkerd** [comment]
This was an issue with validation on IE which has now been fixed in the 4.2 branch which will be released on monday

Cheers

Oli :)
